docs/output.md

```diff
@@ -29,63 +29,43 @@ If CRAM/CRAI files were provided in the samplesheet, they will be converted into
 <details markdown="1">
 <summary>Output files</summary>
 
-- `cram/`
+- `bam/`
   - `<sample>.bam`: BAM file
   - `<sample>.bam.bai`: BAI index file
   </details>
```

Comment:
I don't think this in the output (it was disabled before)

Reply:
Correct. But you can enable it if you would like. I will leave it in for now

Reply:
I don't think so? The publishDir was commented out so I removed it in the last PR, I can re-add it if you want